src/Datalust.Piggy/PiggyModule.cs

```diff
@@ -1,17 +1,26 @@
-using System.Reflection;
+using System;
 using Autofac;
 using Datalust.Piggy.Cli;
+using Datalust.Piggy.Cli.Commands;
 
 namespace Datalust.Piggy
 {
-    class PiggyModule : Autofac.Module
+    public class PiggyModule : Module
```

Comment:
May need to be `internal` as the package is consumable as a library; in a library import this will become visible to consumers.

(`<InternalsVisibleTo>` or the attribute will let tests see the type when internal.)

Reply:
Good suggestion! 👍 

I've added the following code to Datalust.Piggy.csproj:

```
<ItemGroup>
    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
        <_Parameter1>Datalust.Piggy.Tests</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```

And set `PiggyModule` back to being implicitly `internal`.